src/pages/TrueSight/index.tsx

```diff
@@ -69,9 +80,74 @@ export default function TrueSight({ history }: RouteComponentProps) {
     }
   }, [history, tab])
 
+  const isChrome = checkChrome()
+  const theme = useTheme()
+
+  const tooltip = useMemo(() => {
+    if (!isChrome)
+      return t`If you would like to subscribe to notifications, please use Google Chrome. Other browsers will be supported in the future`
+
+    if (subscribe) {
+      return t`Unsubscribe to stop receiving notifications on latest tokens that could be trending soon!`
+    } else return t`Subscribe to get notifications on the latest tokens that could be trending soon!`
+  }, [isChrome, subscribe])
+
+  const handleSubscribe = async () => {
```

Comment:
Can you move all these logics which related to notification to a react hook. somethings like:
```
const {isChrome, subscribe, handleSubscribe, handleUnsubscribe} = useNotificationSubscribe()
```